src/main/scala/uk/gov/nationalarchives/consignmentexport/GraphQlApi.scala

```diff
@@ -8,9 +8,7 @@ import cats.implicits._
 import uk.gov.nationalarchives.tdr.{GraphQLClient, GraphQlResponse}
 import graphql.codegen.GetConsignmentExport.{getConsignmentForExport => gce}
 import uk.gov.nationalarchives.tdr.keycloak.KeycloakUtils
-import graphql.codegen.GetFiles.{getFiles => gf}
 import graphql.codegen.UpdateExportLocation.{updateExportLocation => uel}
-import graphql.codegen.GetOriginalPath.{getOriginalPath => gop}
```

Comment:
Can these API endpoints be removed now, to encourage other clients to use the `consignment` field?

I'm particularly keen to remove `getFiles` if possible, because it does some antivirus filtering which I think is a big opaque: https://github.com/nationalarchives/tdr-consignment-api/blob/79db238258f8597ea5292bda8e6e201b407fde12/src/main/scala/uk/gov/nationalarchives/tdr/api/service/FileService.scala#L48

We'll have to do that filtering at some point once we've worked on the failure paths, but I think it would be useful to make it clearer in the API. (Either by adding a filter field in the request, or by adding a field to each file to mark whether it should be included in the export.)

Reply:
I can't find getFiles being used anywhere so we can remove that from the API. GetOriginalPath is being used by the backend checks but we can switch them over to using the consignment query and then get rid of that as well.